File: src/006. Sum square difference/006.c
#include <stdio.h>

int main()
{
    int sum100pow = 0;
    for (int i = 1; i <= 100; ++i)
    {
        sum100pow += i * i;
    }
    int sum100 = 100 * (100 + 1) / 2;
    printf("%d\n", sum100 * sum100 - sum100pow);
}

File: src/006. Sum square difference/006.go
package main

import "fmt"

func main() {
    var sum100pow int
    for i := 1; i <= 100; i++ {
        sum100pow += i * i
    }
    sum100 := 100 * (100 + 1) / 2
    fmt.Println(sum100 * sum100 - sum100pow)
}
